### ISSUE TITLE
Composite primary key - export/import problem

### ISSUE THREAD

**Issue by ebess** [jepiqueau/jeep-sqlite#13]
Hi, nice package. Found some problems regarding validation of the json which can be exported/imported. Assume following json table:

```
{
  "name": "foo_bars_bar",
  "schema": [
    {
      "column": "\"fooId\"",
      "value": "integer NOT NULL"
    },
    {
      "column": "\"barId\"",
      "value": "integer NOT NULL"
    },
    {
      "constraint": "\"FK_d84ae99b331c8b558039a2d1f78\"",
      "value": "FOREIGN KEY (\"fooId\") REFERENCES \"foo\" (\"id\") ON DELETE CASCADE ON UPDATE CASCADE"
    },
    {
      "constraint": "\"FK_8577beb4178aeb72187702c53ab\"",
      "value": "FOREIGN KEY (\"barId\") REFERENCES \"bar\" (\"id\") ON DELETE NO ACTION ON UPDATE NO ACTION"
    },
    {
      "column": "PRIMARY",
      "value": "KEY (\"fooId\", \"barId\")"
    }
  ],
  "indexes": [
    {
      "name": "IDX_d84ae99b331c8b558039a2d1f7",
      "value": "\"fooId\""
    },
    {
      "name": "IDX_8577beb4178aeb72187702c53a",
      "value": "\"barId\""
    }
  ],
  "values": []
}
```

The primary "column" is counted as column and resulting in invalid json without any modifications applied in export/import process. 
https://github.com/jepiqueau/jeep-sqlite/blob/master/src/utils/utils-json.ts#L98

**Comment by jepiqueau** [comment]
@ebess can you try as a constraint
{
"constraint":"\"PK_fooId_barId\"",
"value":"PRIMARY KEY (\"foodId\","\barId"\")"
}
This should work

**Comment by ebess** [comment]
@jepiqueau ok i see but i just exported + try to reimported without any adjustments. so, the exportToJson method has to be corrupted exporting the composite key as a column to json.

**Comment by jepiqueau** [comment]
@ebess may be i missed that one in the export i will see. Does the import works as i suggested. It is the right way to define it as a constraint if yes you may look in the export an propose a PR

**Comment by ebess** [comment]
Yap, that seems to solve the problem but tbh I don't know exactly how to start using the repo locally for developing. But I will take a look

**Comment by jepiqueau** [comment]
@ebess there are plenty of app starter on https://github.com/jepiqueau

**Comment by jepiqueau** [comment]
@ebess This has been fixed in 1.5.6
